docs/getting-started/first-release.md

```diff
@@ -0,0 +1,199 @@
+    ```shell
+    kubectl -n istio-system port-forward svc/istio-ingressgateway 8080:80
+    ```
+    (b) Send requests using the `Host` header:
+    ```shell
+    curl -H 'Host: httpbin.default' localhost:8080 -s -D - | grep -e '^HTTP' -e app-version
+    ```
+
+## Deploy candidate
+
+A candidate can deployed by simply adding a second version to the list of versions comprising the application:
+
+```shell
+cat <<EOF | helm upgrade --install httpbin $CHARTS/release -f -
+environment: deployment-istio
+application: 
+  versions:
+  - metadata:
+      labels:
+        app.kubernetes.io/version: v0
+    image: kennethreitz/httpbin
+  - metadata:
+      labels:
+        app.kubernetes.io/version: v1
+    image: kennethreitz/httpbin
+  strategy: blue-green
+EOF
+```
+
+??? note "About the candidate version"
+    In this tutorial, the candidate image is the same as the one for the primary version. In a real world example, it would be different. The version label (`app.kubernetes.io/version`) can be used to distinguish between versions.
+
+When the second version is deployed and ready, the Iter8 controller automatically reconfigures the routing; the `VirtualService` is updated to distribute traffic between versions based on the weights.
+
+### Verify routing
+
+You can verify the routing configuration by inspecting the `VirtualService` and/or by sending requests as described above. Requests will now be handled equally by both versions.
+
+## Modify weights (optional)
+
+To modify the request distribution between the versions, add a `weight` to each version. The weights are relative to each other.
+
+```shell
+cat <<EOF | helm upgrade --install httpbin $CHARTS/release -f -
+environment: deployment-istio
+application: 
+  versions:
+  - metadata:
+      labels:
+        app.kubernetes.io/version: v0
+    image: kennethreitz/httpbin
+    weight: 30
+  - metadata:
+      labels:
+        app.kubernetes.io/version: v1
+    image: kennethreitz/httpbin
+    weight: 70
+  strategy: blue-green
+EOF
+```
+
+Iter8 automatically reconfigures the routing to distribute traffic between the versions based on the new weights.
+
+### Verify routing
+
+You can verify the routing configuration by inspecting the `VirtualService` and/or by sending requests as described above. Seventy percent of requests will now be handled by the candidate version; the remaining thirty percent by the primary version.
+
+## Promote candidate
+
+The candidate can be promoted by redefining the primary version and removing the candidate:
+
+```shell
+cat <<EOF | helm upgrade --install httpbin $CHARTS/release -f -
+environment: deployment-istio
+application: 
+  versions:
+  - metadata:
+      labels:
+        app.kubernetes.io/version: v1
+    image: kennethreitz/httpbin
+  strategy: blue-green
+EOF
+```
+??? note "What is different?"
+    The version label (`app.kubernetes.io/version`) was updated. In a real world example, the image would also have been updated.
```

Comment:
I think there are two ways to interpret this. At the beginning of the section, you wrote:

> The candidate can be promoted by redefining the primary version and removing the candidate

This implies that the primary version has been changed and the candidate has been removed.

Whereas, this section:

> The version label (`app.kubernetes.io/version`) was updated. In a real world example, the image would also have been updated.

It implies that the candidate version has been changed and the primary has been removed.

I think it would help to reword these sections.

Reply:
reworded for clarity